src/pint/pint_matrix.py

```diff
@@ -607,8 +607,7 @@ def combine_design_matrices_by_param(matrix1, matrix2, padding=0.0):
     for d_param in matrix2.derivative_params:
         if d_param in base_params:
             raise ValueError(
-                "Input design matrix {} has duplicated "
-                "parameters with matrix {}".format(ii, 0)
+                "Unable to combine the two design matrices as they have duplicated parameters."
```

Comment:
Please ensure that the new message informs the user which parameters are duplicated like the old one did.

Reply:
I have changed this to mention `d_param` in the error message.